packages/zoe/test/swingsetTests/zoe/vat-alice.js

```diff
@@ -403,6 +403,7 @@ const build = async (E, log, zoe, issuers, payments, installations, timer) => {
     // remove the liquidity
     const aliceRemoveLiquidityProposal = harden({
       give: { Liquidity: liquidity(10) },
+      want: { TokenA: moola(0), TokenB: simoleans(0) },
```

Comment:
I thought the point of the change was to make this less necessary. If this line is optional, but you're including it for pedagogical reasons, or to test that presence is allowed, please add a comment to the test.

Reply:
I think there's a misunderstanding here. In the current Zoe on master, all keywords will be filled in in the user's proposal before the contract sees it. In this PR, we purposefully don't fill in the user's proposal, so actually, it is more necessary for the user to use the keywords expected by the contract. In this case, the Autoswap contract expects that all three keywords are used when removing liquidity, so Alice must specify TokenA and TokenB, whereas before she didn't. If Autoswap didn't expect all three keywords are used, there would be no need to add this line. 

To put it another way, previously you had to use every keyword in the contract in your proposal, then we had a PR that allowed you to use none of the keywords in your proposal because they would all be filled in for you, and now we are allowing the users to use whatever subset of keywords they want, but we are holding them to that usage and not filling anything in for them in the proposal.

Reply:
Got it. Thanks for explaining. We'll have to make sure to be clear about that, though our users won't have gone through the various transitions, so they only need to understand the end state.